CMakeLists.txt

```
@@ -88,6 +88,14 @@ endif()
 # Whether to enable SSE
 if(CMAKE_SYSTEM_NAME STREQUAL "Emscripten")
   set(FCL_TARGET_SUPPORT_X64_SSE OFF)
+elseif(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.10.0")
+  cmake_host_system_information(RESULT _has_sse QUERY HAS_SSE)
+  cmake_host_system_information(RESULT _has_sse2 QUERY HAS_SSE2)
+  if(_has_sse AND _has_sse2)
+    set(FCL_TARGET_SUPPORT_X64_SSE ON)
+  else()
+    set(FCL_TARGET_SUPPORT_X64_SSE OFF)
+  endif()
 else()
   set(FCL_TARGET_SUPPORT_X64_SSE ON)
```

`VERSION_GREATER_EQUAL` was introduced in cmake 3.7 only. But we can easily invert the logics and use `VERSION_LESS` that is [available in cmake 2.8](https://cmake.org/cmake/help/v2.8.12/cmake.html) as well:
```suggestion
elseif(${CMAKE_VERSION} VERSION_LESS "3.10.0")
  set(FCL_TARGET_SUPPORT_X64_SSE ON)
else() # Decide based on actual host features (only supported since cmake 3.10)
  cmake_host_system_information(RESULT _has_sse QUERY HAS_SSE)
  cmake_host_system_information(RESULT _has_sse2 QUERY HAS_SSE2)
  if(_has_sse AND _has_sse2)
    set(FCL_TARGET_SUPPORT_X64_SSE ON)
  else()
    set(FCL_TARGET_SUPPORT_X64_SSE OFF)
  endif()
```